WEB_SERVICE/WEB/_HIGH/SSL_ALPACA_ATTACK/meta.json

```diff
@@ -0,0 +1,63 @@
+{
+  "title": "ALPACA Attack in SSL/TLS",
+  "short_description": "Cross-protocol confusion attack in SSL/TLS allowing for man-in-the-middle and service impersonation attacks by redirecting traffic between different protocols.",
+  "risk_rating": "high",
+  "references": {
+    "ALPACA Research Paper": "https://alpaca-attack.com/ALPACA.pdf",
+    "Cross-Protocol MITM Attack": "https://var.thejh.net/http_ftp_cross_protocol_mitm_attacks.pdf",
+    "ALPACA Code": "https://github.com/RUB-NDS/alpaca-code"
+  },
+  "privacy_issue": false,
+  "security_issue": true,
+  "categories": {
+    "SOC2_CONTROLS": [
+      "CC_3_2",
+      "CC_3_3",
+      "CC_5_1",
+      "CC_5_2",
+      "CC_6_1",
+      "CC_6_8",
+      "CC_7_1",
+      "CC_7_2"
```

Comment:
can you please check these because the first 4 I don't think they should be there .

Reply:
I double checked, and it seems to me that they should all be there.